lib/src/vector_math/plane.dart

```diff
@@ -6,7 +6,7 @@ part of vector_math;
 
 class Plane {
   final Vector3 _normal;
-  double _constant;
+  double constant;
```

Comment:
Why this change?
It is a public field - visible outside the package, which will hurt dart2js type inference around dynamic call sites.

Reply:
it was a property (see below) – nixing the pointless property is goodness, right?